includes/backup.js

```diff
@@ -215,49 +222,43 @@ function processBatchSet(dbUrl, parallelism, log, batches, ee, start, grandtotal
     }
 
     // do the /db/_bulk_get request
-    var r = {
-      url: dbUrl + '/_bulk_get',
-      qs: { revs: true }, // gets previous revision tokens too
-      method: 'POST',
-      body: payload
-    };
-    client(r, function(err, res, data) {
-      if (err) {
-        ee.emit('error', err);
-        done();
-      } else {
-        request.checkResponseAndCallbackError(res, function(err) {
-          if (err) {
-            if (!err.isTransient) {
-              // Kill the queue for fatal errors
-              q.kill();
-            }
-            ee.emit('error', err);
-            done();
-          } else {
-            // create an output array with the docs returned
-            data.results.forEach(function(d) {
-              if (d.docs) {
-                d.docs.forEach(function(doc) {
-                  if (doc.ok) {
-                    output.push(doc.ok);
-                  }
-                });
-              }
-            });
-            total += output.length;
-            var t = (new Date().getTime() - start) / 1000;
-            ee.emit('received', {
-              batch: thisBatch,
-              data: output,
-              length: output.length,
-              time: t,
-              total: total
-            }, q, logCompletedBatch);
+    // TODO should use built-in _bulk_get, but revs not accepted as part of body
```

Comment:
I don't think we should add complexity to the client to workaround a service problem, better to make the Couch API consistent. If that doesn't happen then we could consider trying to update the nano method to accept both a qs and a body.